internal/ingress/annotations/alias/main.go

```diff
@@ -35,5 +38,13 @@ func NewParser(r resolver.Resolver) parser.IngressAnnotation {
 // Parse parses the annotations contained in the ingress rule
 // used to add an alias to the provided hosts
 func (a alias) Parse(ing *networking.Ingress) (interface{}, error) {
-	return parser.GetStringAnnotation("server-alias", ing)
+	val, err := parser.GetStringAnnotation("server-alias", ing)
+	if err != nil {
+		return []string{}, err
+	}
+
+	aliases := strings.Split(val, ",")
```

Comment:
this is not covering the case where input is `a, b,c       ,d` etc. Splitting with regular expression `,\s*` would cover that.

Reply:
done